lib/ui/pages/room/room.dart

```diff
@@ -44,38 +48,57 @@ class RoomPage extends StatelessWidget {
           ),
         ],
       ),
-      body: Column(
+      body: Stack(
         children: <Widget>[
-          MessageList(),
-          SizedBox(
-            height: AppSpace.xsmall,
-          ),
-          Row(
-            mainAxisSize: MainAxisSize.min,
-            crossAxisAlignment: CrossAxisAlignment.end,
+          Column(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            mainAxisSize: MainAxisSize.max,
             children: <Widget>[
-              const SizedBox(
-                width: AppSpace.small,
-              ),
-              InputMessageTextField(
-                roomTextController: textController,
+              StreamProvider<List<Message>>(
+                create: (_) => roomController.messageList(room.id),
+                initialData: const [],
+                child: MessageList(),
               ),
-              SizedBox(
-                width: AppSpace.xsmall,
-              ),
-              IconButton(
-                onPressed: () async {
-                  await roomController.sendMessage(textController.text);
-                  textController.clear();
-                },
-                icon: Icon(
-                  Icons.send,
-                ),
+              const SizedBox(
+                height: AppSpace.xBig,
               ),
             ],
           ),
-          SizedBox(
-            height: AppSpace.xsmall,
+          Align(
+            alignment: Alignment.bottomCenter,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.end,
+              children: <Widget>[
+                Row(
+                  mainAxisSize: MainAxisSize.min,
+                  crossAxisAlignment: CrossAxisAlignment.end,
+                  children: <Widget>[
+                    const SizedBox(
+                      width: AppSpace.small,
+                    ),
+                    InputMessageTextField(
+                      roomTextController: textController,
+                    ),
+                    SizedBox(
+                      width: AppSpace.xsmall,
+                    ),
+                    IconButton(
+                      onPressed: () async {
+                        await roomController.sendMessage(
+                            textController.text, 'roomId');
```

Comment:
```suggestion
                        await roomController.sendMessage(
                            textController.text,
                            'roomId',
                        );
```
細かいですが、カンマがあるときれいになると思います。
（実装には直接関係ないのでお好みでお願いします^^）